multinode-demo/start_nodes.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+nodes_file=$1
+remote_user=$2
+ssh_keys=$3
+
+oldIFS="$IFS"
+IFS=$'\n' ip_addr_array=($(<$nodes_file))
+IFS="$oldIFS"
```

Comment:
recommend never, ever touching IFS

Reply:
instead of reading the nodes file like this, just have the nodes file written in bash?